Known issue for vSphere IPI upgrades (#26455)
openshift/openshift-docs: release_notes/ocp-4-6-release-notes.adoc

* When deploying an egress router pod in DNS proxy mode, the pod fails to initialize. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888024[*BZ#1888024*])

* When upgrading a cluster from version 4.5 to 4.6 that was installed with

> installer-provisioned infrastructure on vSphere, the upgrade fails if the

Reviewer: Double-check that the first instance of vSphere is "VMware vSphere."

> control plane node IPs change during the upgrade. As a workaround, you must

Reviewer: ```suggestion
control plane node IP addresses change during the upgrade. As a workaround, you must
```

> reserve the control plane node IPs before upgrading to version 4.6. You can do

Reviewer: ```suggestion
reserve the control plane node IP addresses before upgrading to version 4.6.
```

> this by reviewing your DHCP server's documentation for the configuration of

Reviewer: ```suggestion
Review your DHCP server's documentation for the configuration of
```
